catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/event/retrievestatus/DownloadStatusInfoImpl.java

```diff
@@ -34,9 +34,10 @@ public class DownloadStatusInfoImpl implements DownloadStatusInfo {
 
   private static final String UNKNOWN = "UNKNOWN";
 
-  private Map<String, ReliableResourceDownloader> downloaders = new HashMap<>();
+  private Map<String, ReliableResourceDownloader> downloaders =
+      new HashMap<String, ReliableResourceDownloader>();
 
-  private Map<String, String> downloadUsers = new HashMap<>();
+  private Map<String, String> downloadUsers = new HashMap<String, String>();
```

Comment:
If you want to be confident that absolutely nothing can stick around forever, it may be worth making these status holding maps into some sort of cache with time based eviction (from guava or apache commons) with a large enough time eviction that none would get accidentally get removed before they're done. Though I am not as familiar (in recency or content) with this file so it should be only applied if given thorough scrutiny.